suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -88,10 +87,13 @@ export const AccountImportLoadingScreen = ({
                     if (networkSymbol === 'eth') {
                         fetchedAccountInfo.payload.tokens?.forEach(token =>
                             dispatch(
-                                getTokenDefinitionThunk({
-                                    networkSymbol: 'eth',
-                                    chainId: networks.eth.chainId,
-                                    contractAddress: token.contract,
+                                updateTokenFiatRatesThunk({
```

Comment:
Yes it is correct. It is a change that I made (my commit was squashed). Now the all the token definitions are downloaded right after the start of the app.

This Thunk first checks if the definitions are present and valid. If yes it downloads fiat-rates. In case that token definition is not present, it exits the thunk.